frontend/app/(dashboard)/people/[id]/page.tsx

```
@@ -772,12 +795,16 @@ const dividendsColumns = [
-  ) : (
-    <Placeholder icon={CircleCheck}>This investor hasn't received any dividends yet.</Placeholder>
-  );
+  if (isLoading) {
+    return <TableSkeleton columns={4} />;
+  }
+
+  if (!dividends || dividends.length === 0) {
+    return <Placeholder icon={CircleCheck}>This investor hasn't received any dividends yet.</Placeholder>;
+  }
+
+  const table = useTable({ data: dividends, columns: dividendsColumns });
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)